test/notifications/display_test.dart

```diff
@@ -495,6 +534,112 @@ void main() {
         expectedTitle: eg.selfUser.fullName,
         expectedTagComponent: 'dm:${message.allRecipientIds.join(",")}');
     })));
+
+    test('remove: smoke', () => runWithHttpClient(() => awaitFakeAsync((async) async {
+      await init();
+      final message = eg.streamMessage();
+      final data = messageFcmMessage(message);
+      final expectedGroupKey = '${data.realmUri}|${data.userId}';
+
+      check(testBinding.androidNotificationHost.activeNotifications).isEmpty();
+      await receiveFcmMessage(async, data);
+      check(testBinding.androidNotificationHost.activeNotifications).deepEquals(<Condition<Object?>>[
+        conditionActiveNotif(data, 'stream:${message.streamId}:${message.topic}'),
+        conditionSummaryActiveNotif(expectedGroupKey),
+      ]);
+      testBinding.firebaseMessaging.onMessage.add(
+        RemoteMessage(data: removeFcmMessage([message]).toJson()));
+      async.flushMicrotasks();
+      check(testBinding.androidNotificationHost.activeNotifications).isEmpty();
+
+      check(testBinding.androidNotificationHost.activeNotifications).isEmpty();
```

Comment:
nit:
```suggestion
      check(testBinding.androidNotificationHost.activeNotifications).isEmpty();

```

these are close enough together the reader can be expected to remember the previous one :slightly_smiling_face: